src/Deck/Deck.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import './Deck.css'
+
+class Deck extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  
+  render(props) {
+    return (
+      <article className="deck--container">
+        <h2 className="deck--name">{this.props.userDeck.deckName}</h2>
+        <img
+          className="deck--image"
+          src="https://musingsofalifelongnerd.files.wordpress.com/2015/01/mtg-card-back.jpg"
+          alt={this.props.userDeck.deckName}
+        />
```

Comment:
This is just a placeholder image of the back side of a magic card. I would eventually like to make a PR to the data set which adds an array of 'signature cards' for each deck. Then a deck could be represented by images of its signature cards fanned out or stacked together.